kernel/Cargo.toml

```
@@ -63,6 +63,8 @@ reqwest = { version = "0.12.23", default-features = false, optional = true }
 tokio = { version = "1.47", optional = true, features = ["rt-multi-thread"] }
 # both arrow versions below are optional and require object_store
 object_store = { version = "0.12.3", optional = true, features = ["aws", "azure", "gcp", "http"] }
+# TODO: Remove this once https://github.com/apache/arrow-rs/pull/8244 ships
+comfy-table = { version = "=7.1.4", optional = true }
```

sorry one final thing: to actually get `7.1.x` semantics I think we can do:
```suggestion
comfy-table = { version = "~7.1", optional = true }
```